code/modules/vehicles/mecha/equipment/tools/mining_tools.dm

```diff
@@ -167,12 +167,13 @@
 	if(!loc)
 		STOP_PROCESSING(SSfastprocess, src)
 		qdel(src)
-	if(istype(loc, /obj/vehicle/sealed/mecha/working) && scanning_time <= world.time)
```

Comment:
why'd you get rid of the `scanning_time` check?

Reply:
mistake